src/ec/packages/ec-component-footer-standardised/footer-standardised.story.js

```diff
@@ -1,29 +1,265 @@
+  if (data[2].links) {
+    data[2].title = text('sections[2].title', data[2].title, optionalGroupId);
+    data[2].title_class_name = text(
+      'sections[2].title_class_name',
+      data[2].title_class_name,
+      optionalGroupId
+    );
+    data[2].links.forEach((link, index) => {
+      data[2].links[index].link.label = text(
+        `sections[2].links[${index}].link.label`,
+        data[2].links[index].link.label,
+        optionalGroupId
+      );
+      data[2].links[index].link.path = text(
+        `sections[2].links[${index}].link.path`,
+        data[2].links[index].link.path,
+        optionalGroupId
+      );
+    });
+  }
+  if (data[3].links) {
+    // Follow us.
+    data[3].title = text('sections[3].title', data[3].title, optionalGroupId);
+    data[3].title_class_name = text(
+      'sections[3].title_class_name',
+      data[3].title_class_name,
+      optionalGroupId
+    );
+    data[3].list_class_name = text(
+      'sections[3].list_class_name',
+      data[3].list_class_name,
+      optionalGroupId
+    );
+    data[3].links.forEach((link, index) => {
+      data[3].links[index].link.label = text(
+        `sections[3].links[${index}].link.label`,
+        data[3].links[index].link.label,
+        optionalGroupId
+      );
+      data[3].links[index].link.path = text(
+        `sections[3].links[${index}].link.path`,
+        data[3].links[index].link.path,
+        optionalGroupId
+      );
+      data[3].links[index].icon.name = text(
+        `sections[3].links[${index}].icon.name`,
+        data[3].links[index].icon.name,
+        optionalGroupId
+      );
+      data[3].links[index].icon.path = text(
+        `sections[3].links[${index}].icon.path`,
+        defaultSprite,
+        optionalGroupId
+      );
+      data[3].links[index].icon.size = text(
+        `sections[3].links[${index}].icon.size`,
+        data[3].links[index].icon.size,
+        optionalGroupId
+      );
+    });
   }
-});
+  // Related.
+  if (data[4].links) {
+    data[4].title = text('sections[4].title', data[4].title, optionalGroupId);
+    data[2].title_class_name = text(
+      'sections[4].title_class_name',
+      data[4].title_class_name,
+      optionalGroupId
+    );
+    data[4].links.forEach((link, index) => {
+      data[4].links[index].link.label = text(
+        `sections[4].links[${index}].link.label`,
+        data[4].links[index].link.label,
+        optionalGroupId
+      );
+      data[4].links[index].link.path = text(
+        `sections[4].links[${index}].link.path`,
+        data[4].links[index].link.path,
+        optionalGroupId
+      );
+    });
+  }
+  // Classes.
+  if (data[5].links) {
+    data[5].content_before = text(
+      'sections[5].content_before',
+      data[5].content_before,
+      optionalGroupId
+    );
+    data[5].list_class_name = text(
+      'sections[5].list_class_name',
+      data[5].list_class_name,
+      optionalGroupId
+    );
+    data[5].links.forEach((link, index) => {
```

Comment:
link declared, but never used on these foreach functions

Reply:
Eh, but this is the parameter of a forEach:  function(currentValue, index, arr), if you need to access the index you also need the element, and that's what "link" is in this case. No issue from the linter, but i see we go even further :)